src/state/slices/txHistorySlice/selectors.ts

```diff
@@ -119,33 +133,45 @@ export const selectTxIdsBasedOnSearchTermAndFilters = createDeepEqualOutputSelec
   },
 )
 
-export const selectTxsByAssetId = (state: ReduxState) => state.txHistory.txs.byAssetId
-
-type TxHistoryFilter = {
-  assetIds: AssetId[]
-  accountIds?: AccountId[]
-}
+const selectWalletTxIdsByAccountIdAssetId = createSelector(
+  selectWalletAccountIds,
```

Comment:
this `selectWalletAccountIds` is going to become a pattern that will propagate across portfolio related selectors in the next PR towards persistent storage. thankfully, since selectors compose nicely, we can inject this at the top level and the downstream selectors are none the wiser.